src/modules/ens/ens.service.ts

```diff
@@ -124,7 +124,6 @@ export class EnsService implements OnModuleDestroy {
       );
       this.schedulerRegistry.addInterval('ENS Sync', interval);
       this.InitEventListeners();
-      this.syncENS();
```

Comment:
Rather than making a code change here, I think we should try to change at configuration level so that it isn't a hotfix. We could set `ENS_SYNC_ENABLED = false` instead